schema/common/defaults/configuration/kafka-exporter.yml

```diff
@@ -8,10 +8,10 @@ specification:
   config_flags:
     - "--web.listen-address=:9308" # Address to listen on for web interface and telemetry.
     - '--web.telemetry-path=/metrics' # Path under which to expose metrics.
-    - '--verbosity=0'
```

Comment:
I saw that in previous version we have been using '--log.level=info', and in version 1.4.2 there was '--verbosity=0', so how about verbosity/log.level config_flag in version 1.4.0 ?

Reply:
@plirglo correct me if I'm wrong, but from what I remember, none of the flags works in version 1.4.0 😞

Reply:
Correct, those options don't work in v1.4.0 even if they are specified in documentation :(

Reply:
@plirglo WDYT about adding a comment about that for future upgrades?

Reply:
You mean comment in this config file? Maybe, but i think we should consider rather to use alternative for kafka exporter since it looks like this project is not maintained very well. We've got issue to find alternative: [#2801](https://github.com/epiphany-platform/epiphany/issues/2801)

Reply:
There is high probability that it will replaced, but if no, we forget that.

Reply:
IMO `log.level` or `verbosity` isn't something important here. Logs go through systemd-journald so they are rotated. In normal conditions the default level should be fine. When we upgrade version in the future forgetting to add the `verbosity` flag, user will still be able to add it on they own (our validation only requires an array of strings).

Reply:
Adding a note would be faster than writing all these comments 😞

Reply:
@atsikham I'm not a supporter of leaving commented part of code because this might be used in future. W can add this as a known issue with Exporter, WDYT?

Reply:
I'm ok with any case, just see that some other optiona are already commented.

Reply:
Agree, but these options are possible to use in this version.